charts/refinery/templates/deployment-marker-job.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: {{ include "refinery.fullname" . }}-deploy-marker
+  name: {{ include "refinery.fullname" . }}-deploy-marker-{{ randAlphaNum 5 | lower }}
```

Comment:
What happens if the generated value was previously used? Does it care as long as it's not currently running?

Is there a k8s job number or something we could hook into like we get for CI piplines?

Reply:
I tried to use the `generateName` but helm isn't happy with that. This job is suppose to be deleted as soon as it finishes. The randomness for this name doesn't need to be too good.